saltgui/static/stylesheets/page.css

```diff
@@ -101,6 +100,14 @@ table tr td:last-of-type {
   padding-right: 0;
 }
 
+table tr td.starttime,
```

Comment:
what is the reason to apply this to starttime?

Reply:
I was found this useful to shrink also date depending on windows size:
- before
```
2019, Jan 17 23:29:07.065324
```
- after
```
2019, Jan 17 
23:29:07.065324
```

But I can remove it if you don't like it.